sandbox.js

```diff
@@ -60,7 +65,42 @@ function makeVariableTester(vars){
   `
 }
 
+function makeFunctionTester(functs){
+  if(functs.length === 0){
+    return ``;
+  }else{
+    var newArray = new Array();
+
+    for(funct of functs){
+      var name = funct.name;
+      for(test of funct.tests){
+        newArray.push(`
+            var x = ${name}(${test.input});
+            if(x !== ${test.output}){
+              failedTests.push(${name});
```

Comment:
Maybe also add the specific test that failed to failedTests as well as the function name that failed the test.